### ISSUE TITLE
Label needed for "What's new? post in" select on News Feed

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#380]
Area of site: News Feed
URL: https://hcommons-staging.org/activity/

**Description**
The select dropdown (`<select id="whats-new-post-in" name="whats-new-post-in">`) that appears when the "What's new, ___?" is focused does not have a label. 

**Recommendation**

Any one of these should fix the issue:

1. Change the `name` attribute of the `select` to an `id`, refactor the markup of the label from a `<span>` to a `<label>`, and add a `for="whats-new-post-in"` attribute to the label
2. Add a `title` attribute to the select
3. Add an `aria-label` attribute to the select
